web/src/components/schedule/subcomponents/classes_overview/classes_overview.styles.js

```diff
@@ -0,0 +1,37 @@
+import theme from "common/theme";
+import bubbles from "assets/images/bubbles-bw-min.jpg";
```

Comment:
Are we planning on using all 3 files?

Reply:
no, which is part of why this one is in draft. I was gonna test which is most readable

Reply:
would recommend decreasing opacity as well